test/opf.jl

```diff
@@ -642,17 +642,19 @@ end
         result = solve_opf("../test/data/matpower/case6.m", SOCWRConicPowerModel, sdp_solver)
 
         @test result["termination_status"] == OPTIMAL
-        #@test isapprox(result["objective"], 11472.2; atol = 3e0)
-        #@test isapprox(result["objective"], 11451.5; atol = 3e0)
-        @test isapprox(result["objective"], 11473.4; atol = 3e0)
+        # Solvers get a range of answers for this test.
+        #  * Mosek@10 produces 11472.3
+        #  * SCS@3    produces 11470.3
+        @test 11470.2 <= result["objective"] <= 11472.4
```

Comment:
Previously, SCS reported 11473.4. In both cases, Mosek reports 11472.3. SCS's accuracy is ~1e-4, so both values are within tolerance.